src/Canvas.ts

```diff
@@ -515,6 +515,31 @@ export class CanvasForm extends VisualForm {
     
     
     
+    /**
+    * Activate dashed stroke and set dash style. You can customize the segments and offset.
+    * @example `form.dash()`, `form.dash([5, 10])`, `form.dash([5, 5], 5)`
+    * @param segments Dash segments. Defaults to `[5, 5]`. (See [canvas documentation](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash))
+    * @param offset Dash offset. Defaults to 0. (See [canvas documentation]()https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset)
+    */
+    dash( segments:number[]=[5, 5], offset:number=0 ):this {
+      this._ctx.setLineDash(segments);
+      this._ctx.lineDashOffset = offset;
+      return this;
+    }
+    
+    
+    
+    /**
+    * Deactivate dashed stroke.
+    */
+    noDash():this {

```

Comment:
I wonder if it's simpler to overload `dash` to set/unset dash, instead of having an additional `noDash`.

Eg, `dash( segments:PtLike|boolean, offset:number=0 )`

So any falsy values will unset the dash, like `dash(false)` or `dash([])`

Reply:
totally! feels more clean as well